app/src/main/java/com/example/parkingsystem/mvp/contract/ParkingContract.kt

```diff
@@ -0,0 +1,15 @@
+package com.example.parkingsystem.mvp.contract
+
+interface ParkingContract {
+    interface Model {
+        fun getParkingSize(): Int
+    }
+
+    interface Presenter {
+        fun onParkingSizeButtonPressed()
+    }
+
+    interface View {
+        fun showParkingSizeToast(size: Int)
```

Comment:
Maybe this should be showParkingSizeMessage? what if your client decides to use a snack bar instead of a toast?

Reply:
Done 👍